pkg/image/apis/image/docker/scheme/scheme.go

```diff
@@ -0,0 +1,30 @@
+package scheme
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	serializer "k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"github.com/openshift/api/image/docker10"
+	"github.com/openshift/api/image/dockerpre012"
+	"github.com/openshift/origin/pkg/image/apis/image/docker"
+	internaldockerpre012 "github.com/openshift/origin/pkg/image/apis/image/dockerpre012"
+)
+
+var Scheme = runtime.NewScheme()
+var Codecs = serializer.NewCodecFactory(Scheme)
+var ParameterCodec = runtime.NewParameterCodec(Scheme)
+
+func init() {
+	AddToScheme(Scheme)
+}
+
+func AddToScheme(scheme *runtime.Scheme) {
```

Comment:
nit: call this `Install` to match the pull about to merge

Reply:
k